lib/flight_job/commands/create_script.rb

```diff
@@ -32,42 +32,236 @@
 module FlightJob
   module Commands
     class CreateScript < Command
-      def run
-        # Resolves the answers
-        answers = answers_input || begin
-          if $stdout.tty? && stdin_notes?
-            raise InputError, <<~ERROR.chomp
-              Cannot prompt for the answers as standard input is in use!
-              Please provide the answers with the following flag: #{pastel.yellow '--answers'}
-            ERROR
-          elsif $stdout.tty?
-            prompt_answers
-          else
-            msg = <<~WARN.chomp
-              No answers have been provided! Proceeding with the defaults.
+      MULTI_HELP = "(Press ↑/↓/←/→ arrow to scroll, Space/Ctrl+A|R to select (all|rev) and Enter to finish)"
+      SUMMARY = ERB.new(<<~'TEMPLATE', nil, '-')
+        <%= pastel.bold.underline 'SUMMARY' %>
```

Comment:
A blank line or two before `SUMMARY` would help set it apart from the previous interaction.

Reply:
https://github.com/openflighthpc/flight-job/pull/9/commits/eb6e5e64f72d760a987b5e270a912ff33238643f